### PR TITLE
Add HEAD-QA: A Healthcare Dataset for Complex Reasoning

### PR DESCRIPTION
**Pull request by mariagrandury**
HEAD-QA is a multi-choice HEAlthcare Dataset, the questions come from exams to access a specialized position in the
Spanish healthcare system.

### CODE REVIEW

File: datasets/head_qa/head_qa.py [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "name": datasets.Value("string"),
+                    "year": datasets.Value("string"),
+                    "category": datasets.Value("string"),
+                    "qid": datasets.Value("string"),
+                    "qtext": datasets.Value("string"),
+                    "ra": datasets.Value("string"),
```
Comment (lhoestq) [inline datasets/head_qa/head_qa.py:84]: Could this be an int as `aid` ?

**Review by lhoestq — COMMENTED** [review]
Awesome thanks !
And good dataset card as well

I just have one question about the feature types

File: datasets/head_qa/head_qa.py (hunk 1 above)
Comment (mariagrandury) [reply datasets/head_qa/head_qa.py]: Sure, done!

**Comment by mariagrandury** [comment]
Thank you for your review @lhoestq, I've changed the types of `qid` and `ra` and now they are integers as `aid`.

Ready for another review!

**Review by lhoestq — APPROVED** [review]
Thank you !

All good now :)

**Pull request merged**